homeassistant/components/plex/server.py

```diff
@@ -209,8 +209,10 @@ def update_platforms(self):
                 monitored_users.add(new_user)
 
         try:
-            devices = self._plex_server.clients()
-            sessions = self._plex_server.sessions()
+            devices = await self.hass.async_add_executor_job(self._plex_server.clients)
+            sessions = await self.hass.async_add_executor_job(
+                self._plex_server.sessions
+            )
```

Comment:
Your code will be slightly faster if you extract this into a new method which will be scheduled into the executor as 1 job.

```suggestion
            devices, sessions = await self.hass.async_add_executor_job(
                self._fetch_platform_data
            )
```

```python
def _fetch_platform_data(self):
    return (self._plex_server.clients(), self._plex_server.sessions())
```